wire-ios/Wire-iOS/Sources/UserInterface/App Lock/AppLockModule.Router.swift

```diff
@@ -63,7 +68,7 @@ extension AppLockModule.Router: AppLockRouterPresenterInterface {
     private func presentInputPasscodeModule() {
         // TODO: [John] Clean this up.
         // TODO: [John] Inject these arguments.
-        let unlockViewController = UnlockViewController(selfUser: ZMUser.selfUser(), userSession: ZMUserSession.shared())
+        let unlockViewController = UnlockViewController(selfUser: ZMUser.selfUser(), userSession: userSession)
```

Comment:
You can get rid of these todos now.

Reply:
I already took care of that @johnxnguyen